src/component/registry/bpmn-elements-registry.ts

```diff
@@ -178,14 +165,19 @@ export class BpmnElementsRegistry {
    * @param classNames The name of the class(es) to remove from the BPMN element(s)
    */
   toggleCssClasses(bpmnElementIds: string | string[], classNames: string | string[]): void {
-    // TODO duplication with addCssClasses (only the call to removeClassNames or addClassNames or toggleClasses differ)
+    this.prepareClassesUpdate(bpmnElementIds, classNames, this.cssRegistry.toggleClasses);
+  }
+
+  private prepareClassesUpdate(bpmnElementIds: string | string[], classNames: string | string[], callback: (bpmnElementId: string, classNames: string[]) => boolean): void {
```

Comment:
i prefer a name like this, because we don't just "prepare", we do an action ^^
```suggestion
  private updateCssClasses(bpmnElementIds: string | string[], classNames: string | string[], callback: (bpmnElementId: string, classNames: string[]) => boolean): void {
```

Reply:
👍